script.js

```diff
@@ -0,0 +1,112 @@
+const hamburgerBtn = document.querySelector('.hamburger');
+const crossBtn = document.querySelector('.cross-icon');
+const navLinks = document.querySelectorAll('.navLink');
+const djData = [
+  {
+    hidden: false,
+    photo: './cap-detials/benz.jpeg',
+    alt: 'Benzema  Photo',
+    name: 'Karim Benzema',
+    occupation: '48 goals, 15 assists. Won Champions League, La Liga, UEFA Nations League & Supercopa de Espana',
+    description: ' In terms of consistency through the season, number of top-level performances, standing up in the biggest moments possible and final trophy haul, Benzema ticks every box needed to win a Ballon dOr.',
+  },
+
+  {
+    hidden: false,
+    photo: './cap-detials/mane.jpeg',
+    alt: 'Sadio Mane Photo',
+    name: 'Sadio Mane',
+    occupation: '33 goals, five assists. Won Africa Cup of Nations, FA Cup & Carabao Cup.',
+    description: ' His trophy haul, as well as the help of recency bias, should help the Senegal international finish on the Ballon d Or podium,',
+  },
+
```

Comment:
- [ ] Great job generating the "Featured speakers" dynamically with JavaScript. The section. :partying_face::+1:💪🏻 